apps/passport-server/src/services/telegramService.ts

```diff
@@ -353,6 +271,38 @@ export class TelegramService {
       `,
         { parse_mode: "HTML", reply_to_message_id: messageThreadId }
       );
+      const msg = await ctx.reply(`Loading tickets and events...`);
```

Comment:
Viewing the linked events is now part of the `adminhelp` command, not the `/start` that the user sees.